Some more signals moved to new style

amilan opened the pull request:
Hi, 
Here are some more signals adapted to the new style. 
A review is always welcome.

Cheers,
Antonio.

amilan committed:
Change TaurusGraphicsUpdateThread signals.

Change updateView signal to new style signals.

amilan committed:
Merge remote-tracking branch 'upstream/bug-187' into bug-187

amilan committed:
Fix AutoScrollTool signal

Fix baseplot.SIG_ITEMS_CHANGED signal

amilan committed:
Merge remote-tracking branch 'upstream/bug-187' into bug-187

amilan committed:
Fix signals in TaurusModelChooser

Apply new signal style to TaurusModelChooser signals.

amilan committed:
Fix signals in taurusmodellist.py

Adapt signals to new style

amilan committed:
Update logs

Update SIGNALS_TODO.log and WIP.log

NexeyaSGara commented on lib/taurus/qt/qtgui/graphic/taurusgraphic.py:
```diff
@@ -39,6 +39,8 @@
 
 import Queue
 
+from PyQt4 import Qt
```
I think it's better to use 
from taurus.external.qt import Qt

amilan replied:
Me too, in fact I've it removed in my local file ( from taurus.external.qt import Qt is already in the file ) ... I don't know how it ended up here ...

Thanks!

amilan replied:
Ok, I think it's solved in my last addition to this PR.

Thanks for the review!

amilan committed:
Adapt TaurusTrend signals

Move TaurusTrend qt signals to new style

vxgmichel commented on lib/taurus/qt/qtgui/extra_guiqwt/tools.py:
```diff
@@ -215,7 +215,7 @@ def __init__(self, manager, scrollFactor=0.2, toolbar_id=None):
     def register_plot(self, baseplot):
         ToggleTool.register_plot(self, baseplot)
-        self.connect(baseplot, SIG_ITEMS_CHANGED, self.items_changed)
+        baseplot.SIG_ITEMS_CHANGED.connect(self.items_changed)
```
`guiqwt.signals.SIG_ITEMS_CHANGED` is actually `SIGNAL('items_changed(PyQt_PyObject)')`, so I guess it should be `baseplot.items_changed.connect`.

vxgmichel commented on lib/taurus/qt/qtgui/panel/taurusmodelchooser.py:
```diff
@@ -149,6 +149,8 @@ class TaurusModelChooser(TaurusWidget):
     '''
 
     updateModels = Qt.pyqtSignal('QStringList')
+    # (list<str> or QMimeData)
```
`QMimeData`?

amilan replied:
It seems that UpdateAttrs could emit a list of str or QMimeData (as are the possible returns from getListedModels), so I'll update the signal signature, that should be enough. 
Also, this signal is only used for backwards compatibility with the old AttributeChooser ...

cpascual replied:
> this signal is only used for backwards compatibility with the old AttributeChooser ...
> If that is the case, then do not worry about it, since `AttributeChooser` should be removed

vxgmichel commented on lib/taurus/qt/qtgui/plot/taurustrend.py:
```diff
@@ -90,6 +90,8 @@ class TaurusTrendsSet(Qt.QObject, TaurusBaseComponent):
     # disabling)
     droppedEventsWarning = -1
 
+    dataChanged = Qt.pyqtSignal('const QString &')
```
`QString` alone should work as well.

vxgmichel commented on lib/taurus/qt/qtgui/plot/taurustrend.py:
```diff
@@ -582,6 +584,8 @@ class ScanTrendsSet(TaurusTrendsSet):
     """
     DEFAULT_X_DATA_KEY = 'point_nb'
 
+    dataChanged = Qt.pyqtSignal('const QString &')
```
Same here.

vxgmichel commented on lib/taurus/qt/qtgui/plot/taurustrend.py:
```diff
@@ -876,6 +880,8 @@ class TaurusTrend(TaurusPlot):
 
     DEFAULT_MAX_BUFFER_SIZE = 65536  # (=2**16, i.e., 64K events))
 
+    dataChanged = Qt.pyqtSignal('const QString &')
```
Same here.

amilan committed:
Change SIG_ITEMS_CHANGED to items_changed

Change SIG_ITEMS_CHANGED to items_changed

amilan committed:
Update UpdateAttrs signal signature

Update UpdateAttrs signal signature in order to support emiting list of
str or QMimeData.

amilan committed:
Change dataChanged signal signature

dataChanged signature doesn't need to be 'const QString &', 'QString'
will be enough.
Also, pyqtSignature has been changed by pyqtSlot following the new
style.

amilan committed:
Update updateView signal signature

Update signal signature for updateView.

amilan committed:
Remove unnecessary signal and add specific signature

Remove dataChanged signal. It's not needed because it's already
implemented in a parent class: QAbstractItemModel.

Add more specific signature to dataChanged signal declared in
TaurusModelList

amilan committed:
Merge remote-tracking branch 'upstream/bug-187' into bug-187
